### ISSUE TITLE
[Bug] : Minification breaks the aspect-ratio property

### ISSUE THREAD

**Issue by micahlt** [cssnano/cssnano#963]
**Describe the bug**
When run through cssnano, values for the `aspect-ratio` CSS property (such as `4 / 3`) are automatically converted into decimals (`1.3333333`).  This fully breaks `aspect-ratio` in all major browsers.

**To Reproduce**

1. Use the `aspect-ratio` property
2. Run the stylesheet through cssnano
3. Open the result in a browser

**Expected behavior**
I would expect cssnano to recognize instances of this property and leave them as fractions.  

**Screenshots**
If applicable, add screenshots to help explain your problem. (from [cssnano-playground](https://cssnano.co/playground/))

**Desktop (please complete the following information):**
- CSSNANO Version 4.1.10
- Plugins/presets versions

```bash
  System:
    OS: Windows 10 10.0.19041
    CPU: (12) x64 Intel(R) Xeon(R) CPU           W3680  @ 3.33GHz
    Memory: 8.36 GB / 15.99 GB
  Binaries:
    Node: 14.10.1 - C:\Program Files\nodejs\node.EXE
    npm: 6.14.8 - C:\Program Files\nodejs\npm.CMD
  Managers:
    Gradle: 6.6.1 - C:\Gradle\gradle-6.6.1\bin\gradle.BAT
    pip3: 20.1.1 - ~\AppData\Local\Programs\Python\Python38-32\Scripts\pip3.EXE
  Utilities:
    Git: 2.28.0. - C:\Program Files\Git\cmd\git.EXE
  SDKs:
    Windows SDK:
      AllowDevelopmentWithoutDevLicense: Enabled
  IDEs:
    Android Studio: Version  4.0.0.0 AI-193.6911.18.40.6626763
  Languages:
    Bash: 4.4.23 - C:\Program Files\Git\usr\bin\bash.EXE
    Perl: 5.32.0 - C:\Program Files\Git\usr\bin\perl.EXE
    Python: 3.8.5 - C:\Users\Micah\AppData\Local\Programs\Python\Python38-32\python.EXE
  Browsers:
    Chrome: 86.0.4240.183
    Edge: Spartan (44.19041.423.0), Chromium (86.0.622.63)
    Internet Explorer: 11.0.19041.1

+-- cssnano@4.1.10
+-- parcel@1.12.4
| +-- cssnano@4.1.10  deduped
| `-- htmlnano@0.2.6
|   `-- cssnano@4.1.10  deduped
`-- parcel-bundler@1.12.4
  `-- cssnano@4.1.10  deduped

```

**Additional context**
I'm aware browser support for the `aspect-ratio` property is limited, but it would be great to go ahead and support it.

**Comment by anikethsaha** [comment]
Thanks for the issue. 

Can you please share the link from cssnano playground with your code block

[This is working file](https://cssnano.co/playground/#eyJpbnB1dCI6IkBtZWRpYSAobWF4LWFzcGVjdC1yYXRpbzogNC8zKSB7XHJcbiAgZGl2IHtcclxuICAgIGJhY2tncm91bmQ6ICM5ZmY7ICAvKiBjeWFuICovXHJcbiAgfVxyXG59XHJcblxyXG4uZWwge1xyXG4gIGFzcGVjdC1yYXRpbzogMTYgLyA5O1xyXG59XHJcbi5lbGwge1xyXG4gIGFzcGVjdC1yYXRpbzogNCAvIDM7XHJcbn0iLCJjb25maWciOiIvLyBjc3NuYW5vIGNvbmZpZ1xue1xuICBcInByZXNldFwiIDogXCJkZWZhdWx0XCIsXG59XG4ifQ==)

**Comment by ludofischer** [comment]
Fixed in cssnano 5.0.0-rc.2
